cli/cmd/volume.go

```diff
@@ -82,22 +83,32 @@ var volumeCreateCmd = &cobra.Command{
 }
 
 var volumeStartCmd = &cobra.Command{
-	Use:   "start [flags] <VOLNAME>",
+	Use:   "start <VOLNAME>",
```

Comment:
You can retain [flags], which will be added later for backward compatibility

Reply:
ack